partition/gpt/table.go

```diff
@@ -525,3 +594,35 @@ func (t *Table) GetPartitions() []part.Partition {
 	}
 	return parts
 }
+
+// Verify will attempt to evaluate the headers and repair mis-aligned
```

Comment:
Is it correct that `Verify()` does not actually repair, but just evaluate? I don't see changes here. Can the comment describe what it actually verifies?

Reply:
Yes, `verify()` is just to perform a few checking tasks.

Reply:
Then I think the comment should be updated.